templates/angular/igx-ts/calendar/default/files/src/app/__path__/__name__.component.html

```diff
@@ -0,0 +1,8 @@
+<div igxLayout igxLayoutDir="columns" igxLayoutItemAlign="center">
+	<p>igx-calendar component</p>
```

Comment:
Let's have a placeholder here which clarifies what is the selection mode currently displayed. You car pass the value of the placeholder when setting the extra configuration.